packages/demo/webpack.config.js

```diff
@@ -8,6 +8,9 @@ const prod = mode === 'production'
 module.exports = {
   entry: './src/main.js',
   resolve: {
+    fallback: {
```

Comment:
@taylorjdawson Can we add this to the webpack config in the Core Readme?

Reply:
Good idea!